src/ast/values.rs

```diff
@@ -377,6 +409,12 @@ impl<'a> From<i32> for ParameterizedValue<'a> {
     }
 }
 
+impl<'a> From<&'a [u8]> for ParameterizedValue<'a> {
```

Comment:
Maybe also `From<Vec<u8>>`?

Reply:
It's not possible because of the `From<Vec<T>> for DatabaseValue` I think